ui/components/component-library/index.js

```diff
@@ -32,7 +32,7 @@ export { Tag } from './tag';
 export { TagUrl } from './tag-url';
 export { Text, ValidTag, TextDirection, InvisibleCharacter } from './text';
 export { Input, InputType } from './input';
-export { TextField, TEXT_FIELD_TYPES, TEXT_FIELD_SIZES } from './text-field';
```

Comment:
Because `TEXT_FIELD_TYPES` and `TEXT_FIELD_SIZES ` constants are used throughout the codebase I would suggest deprecating them instead of removing them. Below is doing a search for the constants you can see that there are quite a few results

![Screenshot 2023-08-16 at 10 46 36 AM](https://github.com/MetaMask/metamask-extension/assets/8112138/00c5aa9e-48e8-4579-ae12-5f67399bc955)

That way we don't have to replace every instance in the code base but we are still notifying engineers to cease using them. Here is an example of how we deprecated the `DISPLAY` constant in favor of the `Display` enum. 

https://github.com/MetaMask/metamask-extension/blob/6512cacec837b79703e7a76c949d05f1b0a4f722/ui/helpers/constants/design-system.ts#L324-L340

Instead of link an issue here we can just keep it to:  Help to replace `TEXT_FIELD_TYPES ` with `TextFieldType` by submitting a PR

Reply:
yes, this sounds much better. Will do it.

Reply:
done [`here`](https://github.com/MetaMask/metamask-extension/pull/20477/commits/4318f99409d540d47fb69ebc5cb0f38524469733)

after screenshots:
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/23a4f17e-a98f-4197-b3f6-ea10684df793)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/e460b8da-3f03-4b74-83c7-639792987f92)

Reply:
Excellent!